src/main/java/com/example/poc/mapper/CountryMapper.java

```diff
@@ -0,0 +1,33 @@
+package com.example.poc.mapper;
+
+import com.example.poc.model.entity.Country;
+import com.example.poc.wsdl_classes.ListOfCountryNamesByNameResponse;
+import com.example.poc.wsdl_classes.TCountryCodeAndName;
+import org.springframework.util.CollectionUtils;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CountryMapper {
```

Comment:
Can you make this Mapper more Generic? I saw that all the mappers follow a certain pattern.